tests/subsys/bluetooth/fast_pair/crypto/Kconfig

```diff
@@ -4,27 +4,10 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-source "Kconfig.zephyr"
-
-
-choice BT_FAST_PAIR_CRYPTO_BACKEND
-	prompt "Fast Pair cryptographic backend selection"
-	default BT_FAST_PAIR_CRYPTO_MBEDTLS
-
-config BT_FAST_PAIR_CRYPTO_TINYCRYPT
-	bool "Fast Pair with Tinycrypt cryptographic backend"
-	select TINYCRYPT
-	select TINYCRYPT_SHA256
-	select TINYCRYPT_AES
-	select TINYCRYPT_ECC_DH
+menu "Test configuration"
+rsource "../../../../../subsys/bluetooth/services/fast_pair/fp_crypto/Kconfig.fp_crypto"
```

Comment:
Maybe you could use here an absolute path?
Starting with `source "${NRF_DIR}/`

Reply:
It does not seem to work. That's why I used the relative path here (assuming it's good enough for unit test usecase)

Reply:
Fixed with help of @rakons